mkdocs.yml

```
@@ -0,0 +1,59 @@
+# Project information
+site_name: Arduino Firmware Uploader
+site_description: A firmware uploader/updater for Arduino Boards.
+site_url: https://arduino.github.io/arduino-firmware-uploader/
```

```suggestion
site_url: https://arduino.github.io/FirmwareUploader/
```
https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites
>Unless you're using a custom domain, project sites are available at `http(s)://<username>.github.io/<repository>` or `http(s)://<organization>.github.io/<repository>`.

---

Good to know thanks! I've had some doubts about this piece...